store/confirmCDS.go

```diff
@@ -164,3 +179,62 @@ func (m mongoDB) politicalGeoInfo(loc schema.Location) (PoliticalGeo, error) {
 	}
 	return ret, nil
 }
+func (m mongoDB) ContinuousDataCDSConfirm(loc schema.Location, num int64, lessEqThan int64) ([]schema.CDSScoreDataSet, int, error) {
```

Comment:
```suggestion
func (m mongoDB) ContinuousDataCDSConfirm(loc schema.Location, windowSize int64, latestReportTs int64) ([]schema.CDSScoreDataSet, error) {
```

Reply:
it is not alatestReportTs . it is more like a lessThan